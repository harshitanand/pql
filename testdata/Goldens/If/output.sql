WITH "subquery0" AS (SELECT * FROM "SourceFiles" ORDER BY "LineCount" DESC NULLS LAST, "FileName" ASC NULLS FIRST)
SELECT "FileName" AS "FileName", CASE WHEN coalesce(("LineCount") >= (1000), FALSE) THEN 'Large' ELSE 'Smol' END AS "Size" FROM "subquery0";
