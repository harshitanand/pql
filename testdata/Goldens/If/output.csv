parser_test.go,Large
parser.go,Smol
lex.go,Smol
lex_test.go,Smol
pql.go,Smol
ast.go,Smol
clickhouse_test.go,Smol
golden_test.go,Smol
tokenkind_string.go,Smol
pql_test.go,Smol
